GoedeDoelenHelpen/ClientApp/src/app/user/dashboard/header/header.component.ts

```diff
@@ -13,10 +14,15 @@ export class HeaderComponent implements OnInit {
 
   pages: EventsModel[];
 
-  constructor(private eventsService: EventsService) {}
+  constructor(private eventsService: EventsService, private fbService: FbService ) {}
 
   ngOnInit() {
     this.pages = this.eventsService.getEvents();
   }
 
+  fbAuth() {
+    alert("aaa");
```

Comment:
Dit is geen duidelijke feedback naar de user toe en verwerkt niet de popups zoals wij ze doen.